docs/src/index.md

```diff
@@ -98,11 +99,11 @@ If the installed version does not match the current release, please check the
 [Troubleshooting](@ref old-release) section.
 
 The commands above can also be used to update Trixi.jl. A brief list of notable
-changes to Trixi.jl is available in [`NEWS.md`](https://github.com/trixi-framework/Trixi.jl/NEWS.md).
+changes to Trixi.jl is available in the [`Changelog`](@ref).
```

Comment:
CI errors because of this: https://github.com/trixi-framework/Trixi.jl/actions/runs/12121283510/job/33791875551?pr=2187#step:7:3498

Reply:
Let's see if 112dd7e8ac34798517eb4bec21fa84989a2e8104 fixes this...

Reply:
[Yes](https://github.com/trixi-framework/Trixi.jl/actions/runs/12128909599/job/33816300449?pr=2187).